re issue#20 make accounting for hash algorithm agility explicit (#23)
cose-wg/draft-ietf-cose-merkle-tree-proofs: draft-ietf-cose-merkle-tree-proofs.md

Each specification MUST define how to produce and consume the supported proof types.
See {{sec-rfc-9162-verifiable-data-structure-definition}} as an example.

Where a specification supports a choice of hash algorithm,

> a registration must be made for each individually supported algorithm.

Reviewer: ```suggestion
an IANA registration must be made for each individually supported algorithm.
```

> Where a specification supports a choice of hash algorithm,
> an IANA registration must be made for each individually supported algorithm.

Reviewer 2: ```suggestion
Where a specification supports a choice of hash algorithm, an IANA registration must be made for each individually supported algorithm.
```

For example, to provide for both SHA256 and SHA3_256 with {{RFC9162}},

> both "RFC9162_SHA256" and "RFC9162_SHA3_256" require entries in the relevant registries.

Reviewer 3: ```suggestion
both "RFC9162_SHA256" and "RFC9162_SHA3_256" require entries in the relevant IANA registries.
```

> For example, to provide for both SHA256 and SHA3_256 with {{RFC9162}},
> both "RFC9162_SHA256" and "RFC9162_SHA3_256" require entries in the relevant registries.

Reviewer 2: ```suggestion
For example, to provide for both SHA256 and SHA3_256 with {{RFC9162}}, both "RFC9162_SHA256" and "RFC9162_SHA3_256" require entries in the relevant registries.
```
